tests/test_access.py

```diff
@@ -29,6 +28,191 @@ async def test_dns_caching_by_repeated_calls(address, repeat_call, session):
     when DNS needs to be resolved / looked up from a cache.
     """
     airq = AirQ(address, PASS, session, timeout=5)
-    await airq.get(SUBJECT)
+
+    await airq.get("ping")
     if repeat_call:
-        await airq.get(SUBJECT)
+        await airq.get("ping")
+
+
+@pytest.mark.asyncio
+async def test_blink(session):
+    """Test the /blink endpoint and whether it returns the device ID."""
+    airq = AirQ(IP, PASS, session, timeout=5)
+    device_id = await airq.blink()
+
+    assert re.fullmatch("[0-9a-f]+", device_id) is not None
+
+@pytest.mark.asyncio
+async def test_device_name(session):
+    """Test getting and setting the device name."""
+    airq = AirQ(IP, PASS, session, timeout=5)
+    previous_device_name = await airq.get_device_name()
+
+    new_device_name = "just-testing"
+    await airq.set_device_name(new_device_name)
+
+    device_name_after_setting = await airq.get_device_name()
+
+    await airq.set_device_name(previous_device_name)
+    device_name_after_resetting = await airq.get_device_name()
+
+    assert device_name_after_setting == new_device_name
+    assert device_name_after_resetting == previous_device_name
+
+
+@pytest.mark.asyncio
+async def test_log(session):
+    """Test getting the log. It should be a list."""
+    airq = AirQ(IP, PASS, session, timeout=5)
+    log = await airq.get_log()
+
+    assert type(log) == list
+
+
+@pytest.mark.asyncio
+async def test_config(session):
+    """Test getting the config. It should be a big dictionary."""
+    airq = AirQ(IP, PASS, session, timeout=5)
+    config = await airq.get_config()
+
+    keys_expected = {"HotspotChannel", "TimeServer", "cloudUpload", "id", "logging", "sensors"}
+    keys_found = set(config.keys())
+
+    assert type(config) == dict
+    assert len(config) > 40
+    assert len(keys_expected.difference(keys_found)) == 0
+
+
+@pytest.mark.asyncio
+async def test_possible_led_themes(session):
+    """Test getting the possible LED themes."""
+    airq = AirQ(IP, PASS, session, timeout=5)
+    possible_led_themes = await airq.get_possible_led_themes()
+
+    expected = {"standard", "CO2", "VOC", "PM1", "Noise"}
```

Comment:
According to @dl2080 
the minimal expected set should be
```suggestion
    expected = {"standard", "standard (contrast)", "VOC", "Humidity"}
```
That is the bare PCB with only soldered sensors.
Let me know if I misunderstood your intention here.

Reply:
You understood correctly. I wanted a subset of the minimal available values, so this test would work with any device.
I was orienting by the manual stating the following, thinking this was the minimum:

> The LED theme may be chosen from the following list:
>
> `['standard','co2_covid19','CO2','VOC','CO','PM1','PM2.5','PM10','Noise']`